packages/dd-trace/src/plugins/util/test.js

```diff
@@ -140,3 +147,51 @@ function getTestSuitePath (testSuiteAbsolutePath, sourceRoot) {
 
   return testSuitePath.replace(path.sep, '/')
 }
+
+const POSSIBLE_CODEOWNERS_LOCATIONS = [
+  'CODEOWNERS',
+  '.github/CODEOWNERS',
+  'docs/CODEOWNERS',
```

Comment:
Could this exist in addition to `CODEOWNERS` at the root?

Reply:
they don't mention anything at https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners. I'd assume one of them is ignored